src/main/java/com/bvengo/simpleshulkerpreview/ShulkerSizeExt.java

```diff
@@ -0,0 +1,5 @@
+package com.bvengo.simpleshulkerpreview;
+
+public interface ShulkerSizeExt {
+  public int getInvSize();
```

Comment:
Please rename to `getInventorySize`

Reply:
This was deliberate to match the `getInvStackList()` and `setInvStackList(DefaultedList<ItemStack> list)` methods of `ShulkerBoxBlockEntity.class`. Will do as requested though